fiat-api/src/main/java/com/netflix/spinnaker/config/FiatAuthenticationConfig.java

```diff
@@ -38,6 +29,16 @@
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
```

Comment:
Any reason to split the imports here? For consistence, I'd rather them be kept together